src/admin/class-sign-in-with-google-admin.php

```diff
@@ -768,7 +787,7 @@ protected function find_by_email_or_create( $user_data ) {
 			return $user;
 		}
 
-		$user_pass    = wp_generate_password( 12 );
+		$user_pass    = wp_generate_password( get_option( 'siwg_password_length', 12 ) );
```

Comment:
Hey @ttodua as you mentioned in [your comment](https://github.com/tarecord/sign-in-with-google/pull/84#issuecomment-1019450129), let's just add the filter and not include the setting.

I only want to provide settings for the majority of users and I think this filter will be used by a few developers—so we don't need the extra field in the settings page.

Reply:
ok, as you decide! 
also, if you feel better that way, you can edit the PR on the fly when merging, so you don't wait on my edit. 
otherwise, I have to get to workspace by tomorrow and can amend then.

Reply:
also, I've never been requested a 'review' for PR, so I am not sure if you are waiting for me to change a status (or do whatever action).